### PR TITLE
DC-3562 Integration tests for `DataApiV1Sdk.get`

### PR DESCRIPTION
**Pull request by oleksii-symon-corva-ai**
### Rationale
`DataApiV1Sdk.get` need integrations tests to be reliable.

### Changes
Added integration tests for `DataApiV1Sdk.get` method.

[JIRA ticket](https://corvaqa.atlassian.net/browse/DC-3562)

#### TODO
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,16 +39,20 @@ test: up-cache unit-tests integration-tests down-cache
...
 ## integration-tests: Run integration tests.
 .PHONY: integration-tests
 integration-tests: export CACHE_URL = redis://localhost:6379
-integration-tests: test_path = tests/integration
+integration-tests: export PROVIDER =
+integration-tests: export TEST_DATASET =
+integration-tests: export API_ROOT_URL = https://platform.localhost.ai
+integration-tests: export DATA_API_ROOT_URL = https://data.localhost.ai
+integration-tests: export TEST_API_KEY =
+integration-tests: export TEST_BEARER_TOKEN =
```
Comment (finomayato) [inline Makefile:53]: why not make it inline via `&&`?

**Review by finomayato — COMMENTED** [review]
Somehow we get so many changes in tests/makefiles, at the same time we changed so little in an actual app. Why so?

File: Makefile [hunk 2]
```diff
@@ -39,16 +39,20 @@ test: up-cache unit-tests integration-tests down-cache
 
 ## unit-tests: Run unit tests.
 .PHONY: unit-tests
-unit-tests: test_path = tests/unit
 unit-tests:
-	@coverage run -m pytest $(test_path)
+	@coverage run -m pytest tests/unit
```
[the remark this answers is not in the document]
Comment (oleksii-symon-corva-ai) [reply Makefile:43]: My bad, shouldnt have changed makefile in this pr. Rolling back

File: Makefile (hunk 1 above)
Comment (oleksii-symon-corva-ai) [reply Makefile:55]: good question, made env variable inline

**Comment by oleksii-symon-corva-ai** [comment]
> Somehow we get so many changes in tests/makefiles, at the same time we changed so little in an actual app. Why so?

`DataApiV1Sdk.get` method was already there, it lacked integration tests. This pr adds integration tests.

@finomayato

**Review by ville — APPROVED** [review]

**Pull request merged**